tests/test_physics.py

```diff
@@ -1710,7 +1710,7 @@ def test_rigid_constraints():
             habitat_sim.physics.RigidConstraintType.PointToPoint
         )
         constraint_id_2 = sim.create_rigid_constraint(constraint_settings_2)
-        observations += simulate(sim, 5.0, produce_debug_video)
+        observations += simulate(sim, 7.0, produce_debug_video)
```

Comment:
@aclegg3 Due to compiler changes the assert on L1719 was failing. Cranking up situation time seemed to fix that.